packages/core/content-manager/server/src/history/services/__tests__/history.test.ts

```diff
@@ -95,136 +122,162 @@ describe('history-version service', () => {
-    expect(createMock).toHaveBeenCalledWith({
-      data: {
-        ...historyVersionData,
-        createdBy: userId,
-        createdAt: fakeDate,
-      },
+      await historyService.createVersion(historyVersionData);
+      expect(createMock).toHaveBeenCalledWith({
+        data: {
+          ...historyVersionData,
+          createdBy: userId,
+          createdAt: fakeDate,
+        },
+      });
     });
-  });
 
-  it('creates a history version without any author', async () => {
-    jest.useFakeTimers().setSystemTime(fakeDate);
+    it('creates a history version without any author', async () => {
+      jest.useFakeTimers().setSystemTime(fakeDate);
 
-    const historyVersionData = {
-      contentType: 'api::article.article' as UID.ContentType,
-      data: {
-        title: 'My article',
-      },
-      locale: 'en',
-      relatedDocumentId: 'randomid',
-      schema: {
-        title: {
-          type: 'string',
+      const historyVersionData = {
+        contentType: 'api::article.article' as UID.ContentType,
+        data: {
+          title: 'My article',
         },
-      },
-      status: null,
-    };
+        locale: 'en',
+        relatedDocumentId: 'randomid',
+        schema: {
+          title: {
+            type: 'string',
+          },
+        },
+        status: null,
+      };
 
-    mockGetRequestContext.mockReturnValueOnce(null as any);
+      mockGetRequestContext.mockReturnValueOnce(null as any);
 
-    await historyService.createVersion(historyVersionData);
-    expect(createMock).toHaveBeenCalledWith({
-      data: {
-        ...historyVersionData,
-        createdBy: undefined,
-        createdAt: fakeDate,
-      },
+      await historyService.createVersion(historyVersionData);
+      expect(createMock).toHaveBeenCalledWith({
+        data: {
+          ...historyVersionData,
+          createdBy: undefined,
+          createdAt: fakeDate,
+        },
+      });
     });
   });
 
-  it('should create a cron job that runs once a day', async () => {
-    // @ts-expect-error - this is a mock
-    const mockScheduleJob = scheduleJob.mockImplementationOnce(
-      jest.fn((rule, callback) => callback())
-    );
-
-    await historyService.bootstrap();
+  describe('findVersionPage', () => {
+    it('should contain a meta object with unknown attributes', async () => {
+      const { results } = await historyService.findVersionsPage({
+        contentType: 'api::article.article',
+        documentId: 'randomid',
+        locale: 'en',
+      });
 
-    expect(mockScheduleJob).toHaveBeenCalledTimes(1);
-    expect(mockScheduleJob).toHaveBeenCalledWith('0 0 * * *', expect.any(Function));
+      expect(results[0].meta).toEqual({
+        unknownAttributes: {
+          added: {
+            newField: { type: 'string' },
+            renamed: { type: 'string' },
+          },
+          removed: {
+            title: { type: 'string' },
+          },
+        },
+      });
+    });
```

Comment:
ℹ️ This is the only test I added and associated mocks. Sorry should have done a separate commit for organizing everything into describes.